src/components/Timer/subcomponents/TimerView.tsx

```diff
@@ -89,10 +90,25 @@ export default function TimerView(props: {
     }
   };
 
+  const onMinutesKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+    if (event.key === "ArrowRight" || event.key === "Enter") {
```

Comment:
This means that I can't move my cursor right in the input field, which I find very annoying.

Reply:
same reason as above – either every field is treated like one complete number without cursor positions, or i’ll have to do stuff like "if cursor position === end of input field" so it’s like one textbox. i’ll think about it